tests/e2e/configurer/chain/commands.go

```diff
@@ -25,12 +25,12 @@ func (n *NodeConfig) CreatePool(poolFile, from string) uint64 {
 	_, _, err := n.containerManager.ExecTxCmd(n.t, n.chainId, n.Name, cmd)
 	require.NoError(n.t, err)
 
-	path := "osmosis/gamm/v1beta1/num_pools"
+	path := "osmosis/swaprouter/v1beta1/num_pools"
```

Comment:
We should use the deprecated path in e2e, at least for now.

The reason is that e2e runs before the upgrade and after the upgrade.

So before upgrade, there are no swaprouter queries existing yet. As a result, e2e fails

Reply:
makes sense to me